packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxFlyoutMenu/OnyxFlyoutMenu.ct.tsx

```diff
@@ -65,3 +66,63 @@ test("should open on click", async ({ page, mount }) => {
     await expect(item).toBeEnabled();
   }
 });
+
+test("should open on hover", async ({ page, mount }) => {
+  // ARRANGE
+  const component = await mount(TestWrapperCt, {
+    props: { label: "Choose application language", trigger: "hover" },
+  });
+  const menu = page.locator("ul");
```

Comment:
```suggestion
  const menu = page.getByLabel("Choose application language")
```

Can't we use the label instead?